### DISCUSSION TITLE
詢問-於留言板的統計次數

### DISCUSSION THREAD

**Discussion by pocper1** [1101-GS3073-C/Python-Community-GS3073-C#183] [Q&A]
想詢問老師，留言板的統計次數什麼時候會公布呢~

**Comment by wh20502** [comment]
這個EXCEL表單是否有幫助到你！一起加油吧
https://docs.google.com/spreadsheets/d/1iGcoan7oel4m9HbvGS9buo-iw_CTn96rkNUIbUPWaNA/edit#gid=0
